### ISSUE TITLE
Remove fixed vue version requirement

### ISSUE THREAD

**Issue by DispatchCommit** [hCaptcha/vue-hcaptcha#16]
https://github.com/hCaptcha/vue-hcaptcha/blob/master/package.json#L25

Why is vue being restricted to a specific patch version? This is unnecessarily strict and is conflicting with the latest release of nuxt.js for me because they have moved from `vue@2.6.11` to `vue@2.6.12` and as a result I get errors attempting to build my site.

```
Error:

Vue packages version mismatch:

- vue@2.6.11 (D:\bitwave\node_modules\vue\dist\vue.runtime.common.js)
- vue-template-compiler@2.6.12 (D:\bitwave\node_modules\vue-template-compiler\package.json)
```

```
$ npm ls vue
+-- @hcaptcha/vue-hcaptcha@0.2.2
| `-- vue@2.6.11
`-- nuxt@2.14.4
  +-- @nuxt/builder@2.14.4
  | `-- @nuxt/vue-app@2.14.4
  |   `-- vue@2.6.12
  `-- @nuxt/core@2.14.4
    `-- @nuxt/vue-renderer@2.14.4
      `-- vue@2.6.12

npm ls vue-template-compiler
`-- nuxt@2.14.4
  `-- @nuxt/builder@2.14.4
    `-- @nuxt/vue-app@2.14.4
      `-- vue-template-compiler@2.6.12
```

**Comment by e271828-** [comment]
@DispatchCommit PRs welcome :)

**Comment by DispatchCommit** [comment]
@e271828- Love it, PR done ❤

**Comment by DispatchCommit** [comment]
I've also submitted a followup PR that would move the vue.js dependency to dev dependencies so that this would not be an issue in the future. https://github.com/hCaptcha/vue-hcaptcha/pull/18

I tested a few scenarios with both PR's and haven't run into any errors with either, so should be fine, but the repo devs would know better than I. At a minimum, I wanted to propose the idea and potential solution to decouple the vue.js version from the package.

Hope that helps!
